xml/System/Type.xml

```diff
@@ -7391,7 +7391,7 @@ The <xref:System.Type.GetProperties%2A> method does not return properties in a p
         <Parameter Name="typeName" Type="System.String" />
       </Parameters>
       <Docs>
-        <param name="typeName">The assembly-qualified name of the type to get. See <see cref="P:System.Type.AssemblyQualifiedName" />. If the type is in the currently executing assembly or in Mscorlib.dll, it is sufficient to supply the type name qualified by its namespace.</param>
+        <param name="typeName">The assembly-qualified name of the type to get. See <see cref="P:System.Type.AssemblyQualifiedName" />. If the type is in the currently executing assembly or in mscorlib.dll/System.Private.CoreLib.dll, it is sufficient to supply the type name qualified by its namespace.</param>
```

Comment:
How about something like this:
```suggestion
        <param name="typeName">The assembly-qualified name of the type to get. See <see cref="P:System.Type.AssemblyQualifiedName" />. If the type is in the currently executing assembly or in System.Private.CoreLib.dll (in .NET Core) or in mscorlib.dll (in .NET Framework), it is sufficient to supply the type name qualified by its namespace.</param>
```

Reply:
I like this suggestion. @NextTurn Once you accept this suggestion, we'll :shipit:

Reply:
Shall we just use "the core library"? *System.Private.CoreLib* seems to be a runtime-specific internal concept.

Reply:
@NextTurn I don't think we should call this "the core library" that may just introduce more confusion with the term "core." I do think though that the suggestion should say `(in .NET Core or .NET 5)` though 😄

Reply:
@Thraka We were discussing this in linked issue: <https://github.com/dotnet/dotnet-api-docs/issues/3387> The term "Core assembly" is used several times in .NET public API to describe assembly that contains built-in types: [CompilerParameters.CoreAssemblyFileName](https://docs.microsoft.com/en-us/dotnet/api/system.codedom.compiler.compilerparameters.coreassemblyfilename?view=netframework-4.8), [MetadataLoadContext.CoreAssembly](https://docs.microsoft.com/en-us/dotnet/api/system.reflection.metadataloadcontext.coreassembly?view=dotnet-plat-ext-3.1). 

"Base library" is not good choice here IMO because in CLI spec terms Base class library is not synonimous to mscorlib. BCL refers to a set of types contained in mscorlib, but not the physical assembly itself (physical mscorlib also contains Runtime infrastructure and Reflection libraries).